mimic/canned_responses/loadbalancer.py

```diff
@@ -231,28 +245,68 @@ def delete_node(store, lb_id, node_id, current_timestamp):
 
         if store.lbs[lb_id]["status"] != "ACTIVE":
             resource = invalid_resource(
-                "Load Balancer '{0}' has a status of {1} and is considered "
+                "Load Balancer '{0}' has a status of '{1}' and is considered "
                 "immutable.".format(lb_id, store.lbs[lb_id]["status"]), 422)
             return (resource, 422)
 
         _verify_and_update_lb_state(store, lb_id,
                                     current_timestamp=current_timestamp)
 
-        if store.lbs[lb_id].get("nodes"):
-            for each in store.lbs[lb_id]["nodes"]:
-                if each["id"] == node_id:
-                    index = store.lbs[lb_id]["nodes"].index(each)
-                    del store.lbs[lb_id]["nodes"][index]
-                    if not store.lbs[lb_id]["nodes"]:
-                        del store.lbs[lb_id]["nodes"]
-                    store.lbs[lb_id].update({"nodeCount": len(store.lbs[lb_id].get("nodes", []))})
-                    return None, 202
-
-        return not_found_response("node"), 404
+        if _delete_node(store, lb_id, node_id):
+            return None, 202
+        else:
+            return not_found_response("node"), 404
 
     return not_found_response("loadbalancer"), 404
 
 
+def delete_nodes(store, lb_id, node_ids, current_timestamp):
+    """
+    Bulk-delete multiple LB nodes.
+    """
+    if not node_ids:
+        resp = {
+            "message": "Must supply one or more id's to process this request.",
+            "code": 400}
+        return resp, 400
+
+    if lb_id not in store.lbs:
+        return not_found_response("loadbalancer"), 404
+
+    _verify_and_update_lb_state(store, lb_id, False, current_timestamp)
+
+    if store.lbs[lb_id]["status"] != "ACTIVE":
+        resp = {"message": "LoadBalancer is not ACTIVE",
```

Comment:
Just double-checking - is the not active message for bulk actions different than the not active message for singular actions?  You are probably correct since you've also corrected the error message for `delete_node`, but just wanted to make sure.